pil2-stark/.vscode/launch.json

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
I think we should delete or clean this file, most of the things here only apply to the old prover but not necessary for the library library, I've also seen some "rick" stuff here 😅